cmu15122/q-prime
# Change application to run on subpath

Updated the application so that we can serve it from a subdirectory of the URL, defined by a `PUBLIC_URL` variable in `/client/.env`. 

Done so that we can have the app served from `/ohq` on the production server

To test:
- add `PUBLIC_URL` to `/client/.env` and set its value to whatever (such as `/ohq`)
- just click on everything and check that it works (i..e, routes, buttons, services, etc.)
